apps/web-mzima-client/src/app/shared/components/collections/collections.component.ts

```diff
@@ -240,7 +240,12 @@ export class CollectionsComponent extends BaseComponent implements OnInit {
 
     const visibleTo = collectionData.visible_to.value;
     if (visibleTo === 'only_me') {
+      // collectionData.role = ['me'];
+      // collectionData['view_only']['only_me'] = true;
       collectionData.role = ['me'];
```

Comment:
Always delete the code instead of commenting out it. That makes it easier to read and if we want to see past changes, we can always look at the git-history :)